packages/wp-now/README.md

```diff
@@ -34,6 +116,40 @@ To run the unit tests, use the following command:
 nx test wp-now
 ```
 
+### Migrating from Laravel Valet?
+
+If you are migrating from Laravel Valet, you should be aware of the differences it has with `wp-now`:
+
+-   `wp-now` does not require you to install separately WordPress, create database, connect WordPress to that database, create your use or login. All of these steps are packaged into `wp now start` command and are running under the hood;
+-   `wp-now` works across all platforms (Mac, Linux, Windows);
+-   `wp-now` does not support custom domains or SSL;
+-   `wp-now` offers `theme` and `plugin` modes specific to WordPress development;
```

Comment:
This doesn't mean much if I haven't read the other part of the README. How about this:

```suggestion
-   `wp-now` works with WordPress themes and plugins even if you don't have WordPress installed;
```